cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -73,11 +73,19 @@ func (p *plugin) getArgs() []string {
 		command = p.Command
 	}
 
+	args = append(args, command)
+
+	// Include step name as the CI job name for EngFlow
+	drone_step_name := os.Getenv("DRONE_STEP_NAME")
+	if drone_step_name != "" {
+		args = append(args, "--bes_keywords=engflow:CiCdJobName="+drone_step_name)
+	}
```

Comment:
I'm not all that familiar with engflow, or how ubiquitous it is, but would it make sense to gate this with configuration to avoid setting this flag for builds that might not use engflow?

example:
```
  settings:
    engflow_flags: true
```

Reply:
It doesn't hurt anything to have it set and I'd like to thing that anyone who is using this plugin should be getting onto engflow sooner rather than later. If you are aware of non-R&D repos that are using this plugin though then we should talk because engflow is probably only paid for for R&D so would just want to double check that they have the capacity. But otherwise I think anyone in R&D using this plugin should get this flag.